graphql_compiler/schema_merging/utils.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from graphql.type.definition import GraphQLScalarType
+
+
+class SchemaError(Exception):
```

Comment:
There is actually already a `SchemaError` defined in the compiler. It is in `schema_generation/exceptions.py`.

Reply:
Oh interesting, would it make sense to inherit from that `SchemaError` and make my own more specific errors?

Reply:
Or should I rename my base error to something else?

Reply:
Nice catch, @pmantica1! Let's figure out the circumstances in which the built-in `SchemaError` is raised, and if they in spirit match the kinds of problems we may encounter, then subclass from it.